OtripleS.Web.Api.Tests.Acceptance/APIs/Students/StudentsApiTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using OtripleS.Web.Api.Brokers.DateTimes;
+using OtripleS.Web.Api.Models.Students;
+using OtripleS.Web.Api.Tests.Acceptance.Brokers;
+using Tynamix.ObjectFiller;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Acceptance.APIs.Students
+{
+	[Collection(nameof(ApiTestCollection))]
+	public class StudentsApiTests
+	{
+		private readonly OtripleSApiBroker otripleSApiBroker;
+		private readonly DateTimeBroker dateTimeBroker;
+
+		public StudentsApiTests(OtripleSApiBroker otripleSApiBroker)
+		{
+			this.dateTimeBroker = new DateTimeBroker();
+			this.otripleSApiBroker = otripleSApiBroker;
+		}
+
+		private Student CreateRandomStudent()
+		{
+			var filler = new Filler<Student>();
+			filler.Setup()
+				.OnProperty(student => student.BirthDate).Use(this.dateTimeBroker.GetCurrentDateTime())
+				.OnProperty(student => student.CreatedDate).Use(this.dateTimeBroker.GetCurrentDateTime())
+				.OnProperty(student => student.UpdatedDate).Use(this.dateTimeBroker.GetCurrentDateTime());
```

Comment:
Replace all these three lines with something like this:
```filler.Setup()
    .OnType<DateTimeOffset>().Use(GetRandomDateTime());
```
The function for `GetRandomDate` would be something that looks like this:
```csharp
   private static DateTimeOffset GetRandomDateTime() =>
            new DateTimeRange(earliestDate: new DateTime()).GetValue();

```

Reply:
@hassanhabib can you direct this comment to @tobidub's pull request that was opened before mine? this code is from that branch, as I branched off of that branch to make my updates. Otherwise I can make the change here and that PR could be abandoned - let me know

Reply:
@gauriramesh i updated my PR

Reply:
Just to be clear, for the created and updated dates (except for the case of student update of course) you can just get the most recent time from the system, but for any other dates like birth dates, we definitely want to randomize.